xlators/mgmt/glusterd/src/glusterd.h

```diff
@@ -328,20 +328,11 @@ typedef int (*defrag_cbk_fn_t)(glusterd_volinfo_t *volinfo,
                                gf_defrag_status_t status);
 
 struct glusterd_defrag_info_ {
-    uint64_t total_files;
```

Comment:
I must be missing something, as those do seem to be used? num_files_lookedup for example is used in gf_defrag_get_entry()

Reply:
This one (with the same name) is from `struct gf_defrag_info_` in `/xlators/cluster/dht/src/dht-common.h`.